travis_pypi_setup.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python
```

Comment:
If we do add a travis_pypi_setup.py file to this repo (which we may not need to do), it should point to the triage repo and not collate.

Reply:
Yep, I just copied it over because it was in the collate repo. Happy to update it or remove it of course. Having never used it, though, I don't have any opinion on it. I will point out, however, that there appears to be a [travis command line client](https://docs.travis-ci.com/user/deployment/pypi/) that achieves the same thing.